code/game/gamemodes/game_mode.dm

```diff
@@ -271,7 +271,7 @@
 			if(player.client && player.ready)
 				if(!player.mind in drafted || !player.mind in candidates) // Players were getting placed in candidates AND drafted lists.
 					if(player.client.desires_role(role, display_to_user=poll)) // We don't have enough people who want to be antagonist, make a seperate list of people who don't want to be one
-						if(!jobban_isbanned(player, "Syndicate") && !jobban_isbanned(player, role)) //Nodrak/Carn: Antag Job-bans
+						if(!jobban_isbanned(player, "Syndicate") || !jobban_isbanned(player, role)) //Nodrak/Carn: Antag Job-bans
```

Comment:
This needs to stay an &&, otherwise someone banned from syndicate completely but not cult specifically will still be drafted, and vice versa.